CMakeLists.txt

```
@@ -1,55 +1,68 @@
-################################################################################
+# ##############################################################################
 # Preamble
 
 cmake_minimum_required(VERSION 3.17)
 
 project(
   tuv-x
-  VERSION 0.9.0
-  LANGUAGES Fortran CXX C
-)
+  VERSION 0.8.0
+  LANGUAGES Fortran CXX C)
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${PROJECT_SOURCE_DIR}/cmake")
 
-set(CMAKE_USER_MAKE_RULES_OVERRIDE "${PROJECT_SOURCE_DIR}/cmake/set_defaults.cmake")
+set(CMAKE_USER_MAKE_RULES_OVERRIDE
+    "${PROJECT_SOURCE_DIR}/cmake/set_defaults.cmake")
 
 if(NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE "Release" CACHE STRING
-      "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel."
-      FORCE)
+  set(CMAKE_BUILD_TYPE
+      "Release"
+      CACHE
+        STRING
+        "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel."
+        FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
-message ( STATUS "CMake build configuration for tuv-x(${CMAKE_BUILD_TYPE}) ${PROJECT_VERSION}" )
+message(
+  STATUS
+    "CMake build configuration for tuv-x(${CMAKE_BUILD_TYPE}) ${PROJECT_VERSION}"
+)
```

Not sure why this was reformatted but is there a reason the original was not ideal?  All for getting fid of the spaces between `message`, `(`, and `STATUS` though.
```suggestion
message(STATUS "CMake build configuration for tuv-x(${CMAKE_BUILD_TYPE}) ${PROJECT_VERSION}")
```

---

Seems like my code editor reformats files whenever I save them. I was just debugging the `FindLAPACKE` by printing out the include paths found by the function and it reformats the file whenever I delete those print statements and do `:wq`. I will try to turn this off for cmake files.